frontend/packages/dev-console/src/test/pipeline-data.ts

```diff
@@ -3,7 +3,9 @@ import { Pipeline, PipelineRun } from '../utils/pipeline-augment';
 export enum DataState {
   IN_PROGRESS = 'In Progress',
   SUCCESS = 'Completed Successfully',
-  CANCELLED = 'Cancelled',
+  CANCELLED1 = 'Cancelled at stage1',
```

Comment:
You renamed the enum property but didn't [catch all uses](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/utils/__tests__/pipeline-test-data.ts#L247) - you're breaking in the TS build

Reply:
updated